crates/graphql/src/schema/objects/creator.rs

```diff
@@ -30,15 +31,13 @@ impl Creator {
     pub fn attribute_groups(&self, context: &AppContext) -> FieldResult<Vec<AttributeGroup>> {
         let conn = context.db_pool.get()?;
 
-        let metadatas: Vec<String> = metadata_creators::table
-            .select(metadata_creators::metadata_address)
-            .filter(metadata_creators::creator_address.eq(&self.address))
-            .load(&conn)
-            .context("Failed to load metadata creators")?;
-
         let metadata_attributes: Vec<models::MetadataAttribute> = attributes::table
+            .inner_join(
+                metadata_creators::table
+                    .on(attributes::metadata_address.eq(metadata_creators::metadata_address)),
+            )
+            .filter(metadata_creators::creator_address.eq(&self.address))
```

Comment:
does this need a `.unique()`?

Reply:
No because we want to count each occurrence.